include/openssl/ec.h

```diff
@@ -17,6 +17,11 @@
 #ifndef HEADER_EC_H
 #define HEADER_EC_H
 
+#include <limits.h>
+
+#include <bn_utils.h>
+#include <ec_utils.h>
+
 #include <openssl/asn1.h>
 #include <openssl/bn.h>
```

Comment:
Minor point: this can be removed because it's `#include`d in `bn_utils.h`.

Reply:
Removed as suggested.